### ISSUE TITLE
How to migrate your proposed methods to the field of object detection or image enhancement

### ISSUE THREAD

**Issue by zhengqianisme** [Meta-knowledge-Lab/DLB#5]
Can you briefly share what you need to pay attention to when applying your proposed method to the field of object detection or image enhancement? thanks so much

**Comment by yiqings** [comment]
Hi, @zhengqianisme  I didn't follow your questions. I guess you are asking about any possible modifications to tailor the method to object detection or image enhancement.

Well, the first thought that comes into my mind to make DLB task-agnostic is to replace the KLDivergence loss on logits with the MMD loss or L2-loss on the feature map which is extracted from CNN, and since it is not passed into the following downstream-task-related head, this design can be generalized to any task.

If you are asking about the training settings or configurations for the adaption. Honestly, I didn't get an answer, as I haven't implemented those thoughts in these two fields.

**Comment by zhengqianisme** [comment]
thanks！
